src/rezplugins/shell/sh.py

```diff
@@ -130,13 +143,17 @@ def escape_string(self, value, is_path=False):
                 if not txt.startswith("'"):
                     txt = "'%s'" % txt
             else:
-                if is_path:
+                if is_shell_path:
```

Comment:
What's the difference between a path and a shell path? I'm not sure to understand the difference just by looking at this code and without looking inside the functions that are called.